### PR TITLE
Reorder create UUID index

### LINKED ISSUE #4198: v6 upgrade can be lengthy
### Describe the problem

running sql_upgrade using the `development_easy` docker for testing with a pretty mature db with a lot of patients, encounters, lists, insurance, procedure_results, etc.

would like to fine tune the maria innodb setting

> 36bde99b2bc1        mariadb:10.5            "docker-entrypoint.s…"   6 hours ago         Up 6 hours          0.0.0.0:8320->3306/tcp                                               development-easy_mysql_1

Screenshot from 2021-01-26 15-04-31

### PR DESCRIPTION
**Pull request by stephenwaite**
Fixes #4198 

#### Short description of what this resolves:
speeds upgrade

#### Changes proposed in this pull request:
@bradymiller noticed the create index should come before the update of the tables, thank you

### CODE REVIEW

**Comment by stephenwaite** [comment]
maybe i should redo and leave the 6 to 6.1 out so can be cherry picked?

**Comment by bradymiller** [comment]
@stephenwaite , Wouldn't worry about that. I can cherry-pick and sort that out if needed (it creates a conflict that is easy to deal with on command line git)

File: sql/6_0_0-to-6_1_0_upgrade.sql [hunk 1]
```diff
@@ -125,13 +125,13 @@ CREATE UNIQUE INDEX `uuid` ON `insurance_companies` (`uuid`);
 ALTER TABLE `insurance_data` ADD `uuid` binary(16) DEFAULT NULL;
```
Comment (bradymiller) [inline sql/6_0_0-to-6_1_0_upgrade.sql:125]: at the top of this script, lets add the following (will plan to do this on all upgrades until no longer need to; when uuid is built into all of openemr's queries):
```
#IfUuidNeedUpdate patient_data
#EndIf

#IfUuidNeedUpdate form_encounter
#EndIf

#IfUuidNeedUpdate users
#EndIf

#IfUuidNeedUpdateVertical facility_user_ids uid:facility_id
#EndIf

#IfUuidNeedUpdate facility
#EndIf

#IfUuidNeedUpdate immunizations
#EndIf

#IfUuidNeedUpdate lists
#EndIf

#IfUuidNeedUpdateId procedure_order procedure_order_id
#EndIf

#IfUuidNeedUpdateId drugs drug_id
#EndIf

#IfUuidNeedUpdate prescriptions
#EndIf

#IfUuidNeedUpdateId procedure_result procedure_result_id
#EndIf

#IfUuidNeedUpdate ccda
#EndIf
```

**Comment by bradymiller** [comment]
hi @stephenwaite , thanks for doing this and looks good to me; posted some recs above.
out of curiosity, did this speed things up?

**Comment by stephenwaite** [comment]
thanks @bradymiller , yes, really sped things up so much that didn't need that hack of mine, thanks for pointing out the bug

going just revert that 1 file and then bring in the other one in another PR with your recs above

**Comment by sjpadgett** [comment]
I can bet this will make a  huge difference, thanks @stephenwaite

**Pull request merged**